src/frontend/components/property-type/array/edit.tsx

```diff
@@ -13,29 +14,50 @@ type EditProps = Required<EditPropertyPropsInArray>
 
 type ItemRendererProps = {
   onDelete: (event: MouseEvent, property: PropertyJSON) => boolean;
+  index: number;
 }
 
 const ItemRenderer: React.FC<EditProps & ItemRendererProps> = (props) => {
-  const { ItemComponent, property, onDelete } = props
+  const { ItemComponent, property, onDelete, index, record } = props
+  // eslint-disable-next-line no-undef
```

Comment:
Linter says `btoa` is undefined but it's a part of browser's API: https://developer.mozilla.org/pl/docs/Web/API/WindowBase64/btoa